selfcheckout.py

```diff
@@ -36,7 +36,8 @@ def login(userid):
     if response.status_code == 200:
         return Response(response, mimetype="application/json")
     else:
-        return response
+        return Response('Incorrect Login<br>Try Again', 500)
```

Comment:
This should probably be a 401 Response code. See here: https://en.wikipedia.org/wiki/List_of_HTTP_status_codes